homeassistant/components/recorder/history.py

```diff
@@ -649,13 +648,11 @@ def _sorted_states_to_dict(
     # Get the states at the start time
     timer_start = time.perf_counter()
     if include_start_time_state:
-        run = recorder.run_information_from_instance(hass, start_time)
```

Comment:
This call was previously incorrect because it would only look at the current run